stable/nexus/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: nexus
+version: 0.1.0
+description: A Helm chart for sonotype nexus
```

Comment:
A description such as "Sonatype Nexus is open source repository manager" would be better